content/en/docs/setup/independent/high-availability.md

```diff
@@ -1,538 +1,525 @@
-     ```
+{{< note >}}
+**Note**: There are many configurations for load balancers. The following
+example is only one option. Your cluster requirements may need a
+different configuration.
+{{< /note >}}
 
-    It is worth noting that etcd v3.1.12 is the preferred version for Kubernetes v1.10. For other versions of Kubernetes please consult [the changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md).
+1. Create a kube-apiserver load balancer with a name that resolves to DNS.
+
+    In a cloud environment you should place your control plane nodes behind a TCP
+    forwarding load balancer. This load balancer distributes traffic to all
+    healthy control plane nodes in its target list. The health check for
+    an apiserver is a TCP check on the port the kube-apiserver listens on
+    (default value `:6443`).
+
+    It is not recommended to use an IP address directly in a cloud environment.
+
+    The load balancer must be able to communicate with all control plane nodes
+    on the apiserver port. It must also allow incoming traffic on its
+    listening port.
+
+1. Add the first control plane nodes to the load balancer and test the
+   connection:
+
+    ```sh
+    nc -v LOAD_BALANCER_IP PORT
+    ```
+
+    A connection refused error is expected because the API server is not yet
+    running. A timeout is bad and means the load balancer cannot communicate
+    with the control plane node. Reconfigure the load balancer so that it can
+    communicate with the control plane node.
+
+1. Add the remaining control plane nodes to the load balancer target group.
+
+## Stacked control plane nodes
+
+### Bootstrap the first stacked control plane node
+
+1. Create a `kubeadm-config.yaml` template file:
+
+        apiVersion: kubeadm.k8s.io/v1alpha2
+        kind: MasterConfiguration
+        kubernetesVersion: v1.11.0
+        apiServerCertSANs:
+        - "LOAD_BALANCER_DNS"
+        api:
+            controlPlaneEndpoint: "LOAD_BALANCER_DNS:LOAD_BALANCER_PORT"
+        etcd:
+          local:
+            extraArgs:
+              listen-client-urls: "https://127.0.0.1:2379,https://CP0_IP:2379"
+              advertise-client-urls: "https://CP0_IP:2379"
+              listen-peer-urls: "https://CP0_IP:2380"
+              initial-advertise-peer-urls: "https://CP0_IP:2380"
+              initial-cluster: "CP0_HOSTNAME=https://CP0_IP:2380"
+            serverCertSANs:
+              - CP0_HOSTNAME
+              - CP0_IP
+            peerCertSANs:
+              - CP0_HOSTNAME
+              - CP0_IP
+        networking:
+            # This CIDR is a Calico default. Substitute or remove for your CNI provider.
+            podSubnet: "192.168.0.0/16"
+
+
+1.  Replace the following variables in the template with the appropriate
+    values for your cluster:
+
+   * `LOAD_BALANCER_DNS`
+   * `LOAD_BALANCER_PORT`
+   * `CP0_HOSTNAME`
+   * `CP0_IP`
+
+1. Run `sudo kubeadm init --config kubeadm-config.yaml`
```

Comment:
stray sudo left over, should be removed since we are telling users that commands should be run as root

Reply:
I'll remove in the commit I'm finishing now